clientconn.go

```diff
@@ -670,9 +678,9 @@ func (cc *ClientConn) handleResolvedAddrs(addrs []resolver.Address, err error) {
 
 	cc.curAddresses = addrs
 
```

Comment:
@dfawley I updated the following `if else` while doing a rebase. Please take another look. Thanks!